lib/slave/src/AbstractESC.cc

```diff
@@ -1,9 +1,18 @@
-#include "kickcat/Slave.h"
+#include "kickcat/AbstractESC.h"
 
 
 namespace kickcat
 {
-    bool is_valid_sm(std::shared_ptr<AbstractESC> esc, SyncManagerConfig const& sm_ref)
+    void reportError(hresult const& rc)
+    {
+        if (rc != hresult::OK)
+        {
+            printf("\nERROR: %s code %lu\n", toString(rc), static_cast<uint32_t>(rc));
```

Comment:
`lu` might be a problem depending on cpu architecture used. We should consider changing it to use `PRIU64`?

Reply:
Yes, I'm currently fixing all the warning raised by the compiler used for nuttx. The printing troubles are part of the cleanup